workflows/raster/copy.yaml

```
@@ -60,7 +60,7 @@ spec:
       - name: target
         value: 's3://linz-imagery-staging/test/sample_target/'
       - name: include
-        value: '.tiff?$|.json$|.tfw$'
+        value: '.tiff?$|.json$|.tfw$|capture-area.geojson'
```

Should we anchor the regex? As in

```suggestion
        value: '.tiff?$|.json$|.tfw$|^capture-area.geojson$'
```